test/quicer_SUITE.erl

```diff
@@ -734,26 +735,26 @@ tc_conn_controlling_process(Config) ->
       ct:fail("timeout")
   end.
 
-tc_getopt_raw(Config) ->
-  Parm = param_conn_quic_version,
-  Port = 4569,
-  Owner = self(),
-  {SPid, Ref} = spawn_monitor(fun() -> echo_server(Owner, Config, Port) end),
-  receive
-    listener_ready ->
-      {ok, Conn} = quicer:connect("localhost", Port, default_conn_opts(), 1000),
-      {ok, <<1,0,0,0>>} = quicer:getopt(Conn, Parm),
-      {ok, Stm} = quicer:start_stream(Conn, []),
-      timer:sleep(10),
-      {ok, 4} = quicer:send(Stm, <<"ping">>),
-      receive {quic, <<"ping">>, Stm, _, _, _} -> ok end,
-      {error, badarg} = quicer:getopt(Stm, Parm),
-      ok = quicer:close_connection(Conn),
-      SPid ! done,
-      ensure_server_exit_normal(Ref)
-  after 3000 ->
-      ct:fail("listener_timeout")
-  end.
+%% tc_getopt_raw(Config) ->
```

Comment:
Forgotten comment?

Reply:
I am unsure it is time to drop the raw (return in binary format) support so I leave it here, maybe we will add it back.